src/layouts/EditContactUs.jsx

```diff
@@ -308,46 +326,96 @@ const EditContactUs = ({ match }) => {
     )
       return
 
-    const elem = frontMatter[type][source.index]
-    const elemError = errors[type][source.index]
-    const elemDisplay = displaySections[type][source.index]
-    const elemScrollRef = scrollRefs[type][source.index]
-
-    const newFrontMatter = update(frontMatter, {
-      [type]: {
-        $splice: [
-          [source.index, 1], // Remove elem from its original position
-          [destination.index, 0, elem], // Splice elem into its new position
-        ],
-      },
-    })
-    const newErrors = update(errors, {
-      [type]: {
-        $splice: [
-          [source.index, 1], // Remove elem from its original position
-          [destination.index, 0, elemError], // Splice elem into its new position
-        ],
-      },
-    })
-    const newDisplaySections = update(displaySections, {
-      [type]: {
-        $splice: [
-          [source.index, 1],
-          [destination.index, 0, elemDisplay],
-        ],
-      },
-    })
-    const newScrollRefs = update(scrollRefs, {
-      [type]: {
-        $splice: [
-          [source.index, 1],
-          [destination.index, 0, elemScrollRef],
-        ],
-      },
-    })
+    let elem = ""
```

Comment:
It is quite difficult not to adjust the code here because we need to support the nested dnd of the operating hours. Previously such a functionality was not available, so the code here did not account for that. These adjustments would be necessary to allow for that.